src/lightly_train/_task_models/dinov2_ltdetr_object_detection/transforms.py

```diff
@@ -120,8 +124,12 @@ class DINOv2LTDETRObjectDetectionTrainTransformArgs(ObjectDetectionTransformArgs
             label_fields=["class_labels"],
             min_width=0.0,
             min_height=0.0,
-            clip=True,
-            filter_invalid_bboxes=True,
+            **(
+                dict(filter_invalid_bboxes=True)
+                if ALBUMENTATIONS_VERSION_GREATER_EQUAL_2_0_1
+                else {}
+            ),
+            **(dict(clip=True) if ALBUMENTATIONS_VERSION_GREATER_EQUAL_1_4_5 else {}),
```

Comment:
Had to add this because some old albumentations version don't have the `filter_invalid_bboxes` and `clip` arguments.